js/server.js

```diff
@@ -10,11 +10,26 @@ var app = require("express")();
 var server = require("http").Server(app);
 var io = require("socket.io")(server);
 var path = require("path");
+var ipfilter = require('express-ipfilter').IpFilter;
 
 var Server = function(config, callback) {
 	console.log("Starting server op port " + config.port + " ... ");
 
 	server.listen(config.port);
+	if (config.ipWhitelist === undefined) {
```

Comment:
The defaults for the config can be set in: https://github.com/MichMich/MagicMirror/blob/develop/js/defaults.js that might be a cleaner option.